lib/model/database.dart

```diff
@@ -45,6 +46,20 @@ class Accounts extends Table {
   ];
 }
 
+/// The table of the user's chosen settings independent of account, on this
+/// client.
+///
+/// These apply across all the user's accounts on this client (i.e. on this
+/// install of the app on this device).
+@DataClassName('GlobalSettingsData')
+class GlobalSettings extends Table {
```

Comment:
nit: order the class definitions matching https://github.com/zulip/zulip-flutter/pull/1167#discussion_r1978462412